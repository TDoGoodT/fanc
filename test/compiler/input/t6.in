int add(int a, int ab) {
    return a + ab;
}

void main() {
    int num1 = 5;
    int num2 = 10;
    int sum = add(num1, num2);

    print("Sum of 5 and 10:");
    printi(sum);
}
