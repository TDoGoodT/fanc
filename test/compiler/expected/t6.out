Sum of 5 and 10:
15
